categories.json

```
@@ -0,0 +1,383 @@
+{
+  "countries": [
+    "FR"
+  ],
+  "categories": {
+    "amenity": {
+      "name": {
+        "en": "Amenities",
+        "fr": "Services"
```

I would suggest to use a i18n key instead of listing the name in all languages:

```suggestion
        "i18n_key": "amenities",
```

---

Sure, and it was discussed earlier and we were not sure of if keeping it made sense. I changed to only keep english name in there (so everything related to categories start from this file) and it updates `locales/en/json` when changed.